config/shared/locales.json

```diff
@@ -0,0 +1,5 @@
+[
+  { "code": "en", "label": "English" },
+  { "code": "cs", "label": "česky" },
+  { "code": "fi", "label": "suomi" }
+]
```

Comment:
The nice thing about this file is that we could someday in future add languages here that are first visible only in the development environment only before we release them. :-)